example/pythonlib/dependencies/1-pip-deps/build.mill

```diff
@@ -0,0 +1,29 @@
+//// SNIPPET:BUILD
```

Comment:
For example doc/test files that don't inherit from an upstream example (i.e. they don't share the same relative path starting from `pythonlib` or `scalalib`), you don't need to do these `//// SNIPPET:` things. These are only useful if there is an upstream example that you are splicing in customizations into

Reply:
I don't follow what you mean by "upstream example".

Reply:
ok, I've looked throught the ExampleCrossModule and friends implementation a bit more. If I understand correctly, the subclasses of `ExampleCrossModule` would copy an upstream's example code as-is, only replacing snippets. Thus, effectively, comments would be the same, and only code is actually changed.

I can understand the appeal of that in some situations, but it also feels very constrained to enforce the same wording for all examples. What's your recommendation on this @lihaoyi? Since we can't really always translate concepts from python to java, I think it may be easier to stick with plain examples that don't use scalalib as the upstream.

Also, since you created the `ExampleCrossModulePython`, let me know your thoughts @himanshumahajan138

Reply:
I've removed the `ExampleCrossModulePython` trait and all SNIPPETS in pythonlib in my latest commit. I'll merge this if CI passes, but will keep the last commit separate so we can revert if necessary. I'm still curious to hear your opinions.

Reply:
@jodersky sir i think this is not right,
@lihaoyi said that in the case of dependencies we don't inherit any docs from the upstream but in the case of basic we have same upstream docs (for java, kotlin) and the way kotlin extended the base java module is the same way i did for python 
see the docs there are no such docs now why ??? is this educational ???
please resolve this...

Reply:
@jodersky yeah basically it only allows sharing of coarse-grained blocks, though they csn be snything: docs, code, use comments

I'd say if it's too restrictive just drop the sharing for now, we can reconsolidate later if nexessary

Reply:
> see the docs there are no such docs now why ??? is this educational ???

Did I break something? If so that wasn't intentional

Reply:
Actually you removed the `examplecrossmodule` and this leaded to docs removal and more over you removed snippets like build dependency and all 

But why ?

I think you have to undo this for showing docs for basic examples

Reply:
see #4042